docs/MIGRATION_BETWEEN_NAMESPACES.md

```diff
@@ -0,0 +1,24 @@
+Manual CSI migration into another namespace
+---------------------
+It may happen that CSI charts should be moved into another default or non-default namespace. 
+`helm upgrade` is not suitable in this case due to kubernetes objects fields restrictions.
+So we need to delete and install CSI charts, which is described in the steps below.
+
+1. Delete CSI Operator and CSI Deployment charts without removing any CRs and CRDs
+```yaml
+helm delete csi-baremetal
+helm delete csi-baremetal-operator
+```
+2. Reinstall CSI Operator and CSI Deployment charts into another namespace
+```yaml
+helm install csi-baremetal-operator <chart_path> -n $NAMESPACE <other_args>
+helm install csi-baremetal <chart_path> -n $NAMESPACE <other_args>
+```
+3. Wait for all pods to be ready
+```yaml
+watch kubectl get po -n $NAMESPACE
```

Comment:
All pods or csi only? Can we use `-l app=csi-baremetal`?

Reply:
Sure, missed it